api/mraa/aio.hpp

```diff
@@ -47,14 +47,27 @@ class Aio
      *
      * @param pin channel number to read ADC inputs
      */
-    Aio(unsigned int pin)
+    Aio(int pin)
```

Comment:
Not sure this should be bundled into the main commit? @arfoll better in a separate commit? Just in case it breaks something somewhere.